kubemarine/resources/configurations/defaults.yaml

```diff
@@ -481,6 +481,11 @@ plugins:
       image: 'calico/typha:{{ plugins.calico.version }}'
       nodeSelector:
         kubernetes.io/os: linux
+      tolerations:
+        - key: node.kubernetes.io/network-unavailable
+          effect: NoSchedule
+        - key: node.kubernetes.io/network-unavailable
+          effect: NoExecute
```

Comment:
Arrays are hardly redefinable. The user will have to copy-paste these default items plus add its own (or use advanced [feature](https://github.com/Netcracker/KubeMarine/blob/main/documentation/Installation.md#merge-strategy-positioning))
Probably insert these two items directly in python code instead of declaring them in defaults.yaml?

Reply:
I have doubt about it. The declaration in `defaults.yaml` makes those items explicit whereas code implementation makes those items implicit.

Reply:
Agree, it is explicit, but it is explicit for developer, not a user. The user should not go to defaults.yaml, he/she checks the documentation. Also, the [documentation](https://github.com/Netcracker/KubeMarine/blob/main/documentation/Installation.md#calico) says that the option is intended for custom tolerations, but currently it holds the default ones.
> Custom toleration for calico-typha pods

Reply:
Currently, the default values exist, it can't be redefined, and it's not covered by documentation. At least, there is an ambiguous behavior.